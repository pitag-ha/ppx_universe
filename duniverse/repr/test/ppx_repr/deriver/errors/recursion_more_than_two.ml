type t1 = t2 option

and t2 = t3 option

and t3 = t1 option [@@deriving repr]
