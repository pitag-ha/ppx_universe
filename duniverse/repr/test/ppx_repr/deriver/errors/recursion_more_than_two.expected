Fatal error: exception (Failure "Mutually-recursive groups of size > 2 supported")
